app/views/tag/rss.rss.builder

```diff
@@ -1,19 +1,22 @@
-xml.instruct! :xml, :version => "1.0" 
-xml.rss :version => "2.0" do
+xml.rss :version => '2.0', 'xmlns:atom' => 'http://www.w3.org/2005/Atom' do
   xml.channel do
     xml.title "Research tagged '#{params[:tagname]}'"
     xml.description "Open source environmental science research at Public Lab"
     xml.link "https://#{request.host}/feed/tag/"+params[:tagname]+".rss"
 
     @notes.each do |node|
-    
+
       body = node.body
-      body = "<img src='"+node.main_image.path(:default)+"'/><br /> "+node.body if node.main_image
-    
-      xml.item do
-        xml.title       node.title
-        xml.author      node.author.name
-        if node.power_tag('date') != ''
+      uname = node.author.username
+       email = node.author.email
```

Comment:
Oh, sorry - note that I changed this so we don't publish peoples' email addresses without asking them -- would you mind incorporating these changes too? https://github.com/publiclab/plots2/pull/2221/files

Thanks and apologies, i should have noted this earlier!